manager/controllers/app/fybrikapplication_controller_test.go

```diff
@@ -5,28 +5,35 @@ package app
 
 import (
 	"context"
+	"fmt"
 	"os"
 	"time"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	apiv1alpha1 "fybrik.io/fybrik/manager/apis/app/v1alpha1"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const timeout = time.Second * 30
 const interval = time.Millisecond * 100
 
 var _ = Describe("FybrikApplication Controller", func() {
 	Context("FybrikApplication", func() {
+
+		controllerNamespace := getControllerNamespace()
+		fmt.Printf("FybrikApplication: controller namespace " + controllerNamespace)
+
 		BeforeEach(func() {
 			// Add any setup steps that needs to be executed before each test
 			module := &apiv1alpha1.FybrikModule{}
 			Expect(readObjectFromFile("../../testdata/e2e/module-read.yaml", module)).ToNot(HaveOccurred())
+      module.Namespace = controllerNamespace
```

Comment:
Indent issue?

Reply:
Looks ok on my end

Reply:
Unless I'm missing something I believe that it doesn't look right on GitHub view at least. There are also some lint issues detected by the CI checks. A formatter should solve these things automatically.

Reply:
Fixed